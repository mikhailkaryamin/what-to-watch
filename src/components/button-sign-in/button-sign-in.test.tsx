import * as React from 'react';
import {BrowserRouter as Router} from 'react-router-dom';
import renderer from 'react-test-renderer';

import ButtonSignIn from './button-sign-in.tsx';

describe(`button sign in`, () => {
  const wrapper = renderer
    .create(
        <Router>
          <ButtonSignIn />
        </Router>
    ).toJSON();

  test(`should render button sign in`, () => {
    expect(wrapper).toMatchSnapshot();
  });
});
