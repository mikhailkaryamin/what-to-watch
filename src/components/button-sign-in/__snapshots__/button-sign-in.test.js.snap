// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button sign in should render button sign in 1`] = `
<a
  className="user-block__link"
  href="/login"
  onClick={[Function]}
>
  Sign in
</a>
`;
